src/Admin/OptionsMenu.php

```diff
@@ -325,4 +334,19 @@ public function render_screen() {
 		</div>
 		<?php
 	}
+
+	/**
+	 * Adds REST paths to preload.
+	 */
+	public function add_preload_rest_paths() {
```

Comment:
Same question as above.

```suggestion
	protected function add_preload_rest_paths() {
```